repository/src/test/java/org/apache/atlas/repository/store/graph/v2/tags/TagDAOCassandraImplTest.java

```diff
@@ -220,88 +220,6 @@ void testDeleteDirectTag() throws AtlasBaseException {
         );
     }
 
-    @Test
-    void testPutAndGetPropagatedTags() throws AtlasBaseException {
```

Comment:
Removed old tests due to compilation errors. The methods being tested in DAO were removed.